save_sphere.py

```diff
@@ -177,32 +202,31 @@ def record_save_sphere(x_val:int, y_val:int, diag_prog:int, actor:int):
     actor_num = str(actor)
     print("========================")
     newVal = {
-        map_num: {
-            diag_num: {
-                actor_num: {
-                    "x": x_val,
-                    "y": y_val,
-                    "diag": diag_prog
-                }
-            }
-        }
+        map_num: {diag_num: {actor_num: {"x": x_val, "y": y_val, "diag": diag_prog}}}
     }
     if map_num in records.keys():
-        if diag_num in (
-            records[map_num].keys()
-        ):
+        if diag_num in (records[map_num].keys()):
             if actor_num in records[map_num][diag_num]:
-                if records[map_num][diag_num][actor_num]["diag"] != newVal[map_num][diag_num][actor_num]["diag"]:
-                    
-                    records[map_num][diag_num][actor_num]["x"] = newVal[map_num][diag_num][actor_num]["x"]
-                    records[map_num][diag_num][actor_num]["y"] = newVal[map_num][diag_num][actor_num]["y"]
-                    records[map_num][diag_num][actor_num]["diag"] = newVal[map_num][diag_num][actor_num]["diag"]
+                if (
+                    records[map_num][diag_num][actor_num]["diag"]
+                    != newVal[map_num][diag_num][actor_num]["diag"]
+                ):
+
+                    records[map_num][diag_num][actor_num]["x"] = newVal[map_num][
+                        diag_num

```

Comment:
I hate how `black` formats so of this stuff, but it's the price we pay for using `black`. It's better to have it than to not have it.